src/useValidation/useValidation.ts

```diff
@@ -98,6 +112,33 @@ export function useValidation<T>(
     return valid.value
   }
 
+  const reset: ResetMethod = (resetCallback) => {
+    error.value = ''
+    pending.value = false
+    validated.value = false
+
+    if (resetCallback) {
+      pause()
+      try {
+        resetCallback()
+      } finally {
```

Comment:
We probably want to catch the error and `console.error` it. Otherwise we'll swallow errors and could make debugging difficult when using this feature.

Reply:
Hm really? 

I think that'd be the case if there was an empty `catch(e) {}` block here but since there's no catch, an error remains uncaught and should bubble up (-- and the finally block should still run as expected)? 🤔 

I think I'd expect to be able to catch an error from the callback with a try/catch wrapped around `reset`.. but hmm I can verify that behavior real quick tho.

Reply:
oh, no I believe you're correct. I wasn't thinking about this correctly.

Reply:
cool - tweaked this test to make sure it works as expected - [d037165](https://github.com/PrefectHQ/vue-compositions/pull/337/commits/d037165d0a2dd1b43118327a2a663f8cc1ef17cd)

Reply:
TY, that does make it more clear 👍